source/renderer/app/containers/staking/dialogs/DelegationSetupWizardDialogContainer.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+import React, { Component } from 'react';
+import { observer, inject } from 'mobx-react';
+import { defineMessages, intlShape } from 'react-intl';
+import { map } from 'lodash';
+import DelegationSetupWizardDialog from '../../../components/staking/delegation-setup-wizard/DelegationSetupWizardDialog';
+import type { InjectedContainerProps } from '../../../types/injectedPropsType';
+import { formattedWalletAmount } from '../../../utils/formatters';
+
+const messages = defineMessages({
+  learnMoreLinkUrl: {
+    id: 'staking.delegationSetup.intro.step.dialog.learnMore.url',
+    defaultMessage: '!!!https://iohk.zendesk.com/hc/en-us/',
+    description:
+      '"Learn more" link URL on the delegation setup "intro" dialog.',
+  },
+  delegationSetupStep1Label: {
+    id: 'staking.delegationSetup.steps.step.1.label',
+    defaultMessage: '!!!Wallet',
+    description: 'Step 1 label text on delegation steps dialog.',
+  },
+  delegationSetupStep2Label: {
+    id: 'staking.delegationSetup.steps.step.2.label',
+    defaultMessage: '!!!Stake pool',
+    description: 'Step 2 label text on delegation steps dialog.',
+  },
+  delegationSetupStep3Label: {
+    id: 'staking.delegationSetup.steps.step.3.label',
+    defaultMessage: '!!!Delegation',
+    description: 'Step 3 label text on delegation steps dialog.',
+  },
+  delegationSetupStep4Label: {
+    id: 'staking.delegationSetup.steps.step.4.label',
+    defaultMessage: '!!!Activation',
+    description: 'Step 4 label text on delegation steps dialog.',
+  },
+});
+
+type Props = InjectedContainerProps;
+
+type State = {
+  activeStep: number,
+};
+
+const MIN_DELEGATION_FUNDS = 1;
```

Comment:
@tomothespian please move this variable under `source/renderer/app/config/stakingConfig.js` - this file doesn't exist yet...